### ISSUE TITLE
null hypothesis with significant DE

### ISSUE THREAD

**Issue by pedriniedoardo** [cailab-tamu/scTenifoldNet#8]
hello, thank you for developing and maintaining the tool!
I tried to use my dataset for simulating a situation like your outputH0 example (using in X and Y the same input dataset). I am a bit puzzled by the result, as it returns a set of significant DE features (111 genes over a dataset of 6000 genes X 3000 cells).
do you have any idea why is that the case, I would have expected no difference as you showed in the tutorial?

**Comment by dosorio** [comment]
Hi @pedriniedoardo,

Thanks for using scTenifoldNet. Associated P-values are computed based on the distance obtained after the manifold alignment. Just as in differential expression analysis, a cut-off for the distance (I am sure yours are something about d^-15 or so) is needed when the problem is expanded to the entire transcriptome. This approach will reduce the number of false positives obtained in your analysis. 

Please let me know if this solves your issue,

Best wishes,

Daniel

**Comment by pedriniedoardo** [comment]
Hi @dosorio 
thank you for your reply. I just checked and it is like you said indeed!
![image](https://user-images.githubusercontent.com/15876264/162269408-f24a3ae9-0953-41fe-8ade-027b6306fd9e.png)
These are the 4 comparisons I did, and the IMMUNE_CCA__null was the simulation using the same dataset in both X and Y.

![image](https://user-images.githubusercontent.com/15876264/162269688-959c3357-5c7f-4217-8363-aa8f77e1ad88.png)
this is the same data showing also the significant features by padj < 0.05.
clearly, the "significant" one in the null comparison is on a completely different scale in terms of distance...

In this case, I can easily pick a cut-off of distances to focus on the most robust features. But I was wondering whether is there a general recommended cut-off in terms of distance for significance (just like for FC is recommended at least a abs(log2FC)>1)?
thank you again for helping me understand the result!
